pyanaconda/modules/network/nm_client.py

```diff
@@ -51,6 +53,22 @@
 }
 
 
+def get_new_nm_client():
+    """Get new instance of NMClient.
+
+    :returns: an instance of NetworkManager NMClient or None if system bus
+              is not available or NM is not running
+    :rtype: NM.NMClient
+    """
+    if SystemBus.check_connection():
+        nm_client = NM.Client.new(None)
+        if nm_client.get_nm_running():
+            return nm_client
+        else:
+            log.debug("NetworkManager is not running")
+            return None
```

Comment:
It should explicitly return `None` also if the system bus is not available.

Reply:
Addressed in https://github.com/rhinstaller/anaconda/pull/4322